<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle">
      <string>Visual Welcome Center</string>
    </property>
    <property name="toolTipDuration">
      <number>1</number>
    </property>
    <property name="documentMode">
      <bool>false</bool>
    </property>
    <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
    </property>
    <widget class="QStackedWidget" name="vwcStack">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>21</y>
          <width>800</width>
          <height>579</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="toolTip">
        <string>Detect Faces</string>
      </property>
      <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
        <bool>false</bool>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 239, 239);</string>
      </property>
      <property name="currentIndex">
        <number>1</number>
      </property>
      <widget class="QWidget" name="detectFaceWidget">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>21</y>
            <width>800</width>
            <height>579</height>
          </rect>
        </property>
        <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
        </property>
        <widget class="QWidget" name="recordVideoWidget" native="true">
          <property name="geometry">
            <rect>
              <x>85</x>
              <y>30</y>
              <width>630</width>
              <height>500</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="styleSheet">
            <string notr="true"/>
          </property>
        </widget>
      </widget>
      <widget class="QWidget" name="completeRegistrationWidget">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>21</y>
            <width>800</width>
            <height>579</height>
          </rect>
        </property>
        <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
        </property>
        <widget class="QLabel" name="completeRegisterLabel">
          <property name="geometry">
            <rect>
              <x>50</x>
              <y>40</y>
              <width>700</width>
              <height>75</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="toolTipDuration">
            <number>1</number>
          </property>
          <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
            <string>Welcome to the Computer and Information Sciences Department. If you have not registered scan the QR code below in order to complete the registration.</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
        <widget class="QWidget" name="registerPhotoWidget" native="true">
          <property name="geometry">
            <rect>
              <x>100</x>
              <y>225</y>
              <width>200</width>
              <height>200</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="styleSheet">
            <string notr="true"/>
          </property>
        </widget>
        <widget class="QWidget" name="scanQRWidget" native="true">
          <property name="geometry">
            <rect>
              <x>375</x>
              <y>150</y>
              <width>350</width>
              <height>350</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="styleSheet">
            <string notr="true"/>
          </property>
        </widget>
      </widget>
      <widget class="QWidget" name="loginWidget">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>21</y>
            <width>800</width>
            <height>579</height>
          </rect>
        </property>
        <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
        </property>
        <widget class="QLabel" name="loginLabel">
          <property name="geometry">
            <rect>
              <x>300</x>
              <y>50</y>
              <width>300</width>
              <height>50</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="toolTipDuration">
            <number>1</number>
          </property>
          <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
            <string>Welcome to the Computer and Information Sciences Department. If you have not registered scan the QR code below in order to complete the registration.</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
        <widget class="QWidget" name="loginPhotoWidget" native="true">
          <property name="geometry">
            <rect>
              <x>50</x>
              <y>25</y>
              <width>150</width>
              <height>150</height>
            </rect>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="styleSheet">
            <string notr="true"/>
          </property>
        </widget>
        <widget class="QTextBrowser" name="userDetailsText">
          <property name="geometry">
            <rect>
              <x>150</x>
              <y>200</y>
              <width>500</width>
              <height>300</height>
            </rect>
          </property>
        </widget>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>21</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>0</y>
            <width>135</width>
            <height>94</height>
          </rect>
        </property>
        <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
        </property>
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="homeButton"/>
        <addaction name="exitButton"/>
      </widget>
      <widget class="QMenu" name="menuHelp">
        <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
        </property>
        <property name="toolTip">
          <string>Help Menu</string>
        </property>
        <property name="toolTipDuration">
          <number>1</number>
        </property>
        <property name="title">
          <string>Help</string>
        </property>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuHelp"/>
    </widget>
    <action name="actionHome">
      <property name="text">
        <string>Home</string>
      </property>
    </action>
    <action name="actionExit">
      <property name="text">
        <string>Exit</string>
      </property>
    </action>
    <action name="homeButton">
      <property name="text">
        <string>Home</string>
      </property>
      <property name="toolTip">
        <string>Navigate to Home</string>
      </property>
    </action>
    <action name="helpButton">
      <property name="text">
        <string>Help</string>
      </property>
      <property name="toolTip">
        <string>Documentation in progress</string>
      </property>
    </action>
    <action name="exitButton">
      <property name="text">
        <string>Exit</string>
      </property>
      <property name="toolTip">
        <string>Exit Application</string>
      </property>
    </action>
  </widget>
  <resources/>
  <connections/>
</ui>
